mrc-ide/hint
# Mrc-3281 As a user I can see ADR source URL in output zip for input data so I know the source of the files

## Description

This PR adds resource_url to session file manager for onward transmission to hintr for output file. 

To test:

* set `USE_MOCK_MODEL` to false in `run-dependencies`
* Run app as normal without to download step
* download outputFile
* check `info/inputs.csv` file for resource_url


## Type of version change
_Delete as appropriate_

Major - Minor - Patches - None

## Checklist
